donkeycar/utilities/TrackSpeedPlanner/deploy-pi-tornado.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Tornado-based Raspberry Pi deployment script for Path Data Visualizer & Editor
+# Uses Python instead of Node.js for better Pi compatibility
+
+set -e
+
+echo "🐍 Setting up Path Data Visualizer on Raspberry Pi (Python/Tornado)..."
+
+# Clean up any previous installation
+if [ -f "uninstall-pi.sh" ]; then
+    echo "🧹 Cleaning up previous installation..."
+    chmod +x uninstall-pi.sh
+    ./uninstall-pi.sh
+    echo ""
+fi
+
+# Check Python installation
+if ! command -v python3 &> /dev/null; then
```

Comment:
This looks incorrect. We don't need to do anything with the system python. Because we have created our virtual environment already and it usually activated at login. I think this whole install script is obsolete, because our env has tornado installed already.